src/controllers/order/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Body, Tags, Security, ValidateError, SuccessResponse, Response, Controller, Post, Route } from 'tsoa'
+import { Logger } from 'pino'
+
+import { DemandRequest, DemandResponse } from '../../models'
+import { logger } from '../../lib/logger'
+import { BadRequest, NotFound } from '../../lib/error-handler'
+import Database from '../../lib/db'
+import { getMemberByAddress, getMemberBySelf } from '../../lib/services/identity'
+
+const SUBTYPE = 'order'
+const STATE = 'created'
+
+@Route('order')
+@Tags('order')
+@Security('bearerAuth')
+export class order extends Controller {
+  log: Logger
+  db: Database
+  constructor() {
+    super()
+    this.log = logger.child({ controller: '/order' })
+    this.db = new Database()
+  }
+
+  /**
+   * A Member creates a new demand for a order by referencing an uploaded parameters file.
+   * @summary Create a new order demand
+   * @param parametersAttachmentId The attachment's identifier
+   */
+  @Post()
+  @Response<BadRequest>(400, 'Request was invalid')
+  @Response<NotFound>(404, 'Demand was not found')
+  @Response<ValidateError>(422, 'Validation Failed')
+  @SuccessResponse('201')
+  public async createCapacity(@Body() { parametersAttachmentId }: DemandRequest): Promise<DemandResponse> {
+    const [attachment] = await this.db.getAttachment(parametersAttachmentId)
+    if (!attachment) throw new NotFound('attachment')
+
+    const selfAddress = await getMemberBySelf()
```

Comment:
I realised from looking at this that we can use `getMemberSelf()` to get `address` and `alias` at the same time, if you change it to `return member` rather than `member.address` then don't need the second call to identity service to get the owner alias in the `return` block of this function

Reply:
so this line would become something like `const { address, alias } = await getMemberBySelf()`

Reply:
my mistake from when I did `capacity`

Reply:
I rather not do it, due to required updates in other places, I think this could be created as another story same as for db module.